README.md

```diff
@@ -172,19 +180,19 @@ Currently, only json and csv are supported for synchronizing custom files, and t
 
 ## ❓FAQ
 
-### Can't open ChatGPT
+### 1. Can't open ChatGPT
 
 If you cannot open the application after the upgrade, please try to clear the configuration file, which is in the `~/.chatgpt/*` directory.
 
-### Out of sync login status between multiple windows
+### 2. Out of sync login status between multiple windows
```

Comment:
those four elements are in one level and don't have particular orders.

Reply:
sure